### PR TITLE
Changes for 1.09 release

### PR DESCRIPTION
**Pull request by mikecappella**
Version 1.09:
New Converters:
- passwordsafe:       Password Safe (pwsafe) (converter name:
passwordsafe).
- stickypassword:     Sticky Password (converter name:
stickypassword).
- wallet4:            Subsembly Wallet 4X,4W
- truekey:            Intel True Key JSON converter (export supports
logins only)
- keyfinder:          Key Finder (Mac Product Key Finder)

Fixed:
- Clarify SPB Wallet export section in the README file.
- Check for the existence of the export file before passing it onto
the converter.
- Added tag list parameter to help output for --tag option.
- Added --tag option description to the README.
- Handle a possible "version" line in an iCloud keychain export for
keychain converter.
- The roboform converter was not clearing the per-entry variable used
to collect attibutes. Shame.
- The onepif2html converter was outputting unnecessary web form
details for Login items.
- Avoid 'undefined' warning in safeincloud converter.
- In keychain converter, use 'type' = "note" to indicate a secure
note instead of 'desc' whose
value string may be localized (and not 'secure note').
- Updated the README files to reflect 1Passwords new Import dialogs.
- Fixed some CSS and HTML errors in the onepif2html converter.
- The new wallet4 converter was not encoding the language file
correctly (as UTF8).
- Converter keepass2 was in some cases incorrectly handling the
status result of decompressing an
attachment, and in this case, was also prematurely aborting the
import conversion, causing
the inability to create the 1PIF output file.
- Converter keepassx didn't handle timestamps with the value "Never"
when --modified was in use.
- Converter passwordagent updated to support the new XML export
format for versions above 2.6.3.
- Added converter onepif2html2 to the AppleScript Helper temporarily.
It is a temporary converter
producing expanded Secure Notes output, until a satisfactory output
format is determined.
- Updates to the AppleScript Conversion Helper to work correctly on
OS X Sierra.  Due to sandboxing,
the script now requires the convert_to_1p4 folder to be placed on
the Desktop.  This hard-coded
path to the converter script folder simplifies user interaction.

New:
- Added some additional debug code to the keychain converter to help
track down an issue.
- Added password history support to PIF modules.
- Added initial support for Google Apps export of Contacts (vCards).
- The onepif2html converter will accept the 1PIF bundle directory
name (and uses the file data.1pif
contained within).
- Converter keepass2 will store an entry's historic passwords (from
the History element) into
1Password's Previously used passwords list.
- Converter csv now supports a CSV file containing Credit Card and
Membership entries.  Currently
the converter supports Logins, Credit Cards, and Memberships, only
one category per CSV file.
- AppleScript Conversion Helper now accepts additional command line
options.

Changed:
- Updated two 1.08 log notes below: one spelling correction, and one
correcting the location of
function normalize_card_data().
- Tighten username and password regular expressions in roboform
converter.
- Updated the Strawberry Perl links in the README files to the latest
Perl version, and simplified the
command line for installing the required modules.  A new batch
script is used to install the modules
and set the working directory.

Internal Changes:
- Added an assertion to ensure that section entries never contain
null values for key attributes.
- Removed some old, commented-out, no longer used code in the
roboform converter.

### CODE REVIEW

File: convert_to_1p4/README.md [hunk 1]
```diff
@@ -631,6 +682,16 @@ in the command shell.  This only needs to be done once, prior to using this conv
 
 ---
 
+### ● Wallet 4 (4X / 4W)
+
+Launch Wallet 4, and export its database as an HTML file using the `Wallet > Export HTML...` menu.  Navigate to your **Desktop** folder, and in the `Save As` area, enter the file name **pm_export** (note: your file will be named **pm_export.html** after exporting the data).  Click `Save`.  You may now quit Wallet 4.
+
+**Note**: Wallet 4 does not export attached files or images, so they are not available to the converter.  Be sure to export your files from Wallet 4 before you remove it from your system.  A Files or Pictures item in Wallet 4 will be imported as a Secure Note in 1Password, and it contains a reference to the original file name in the notes section.
+
+**Note**: Favorites will not be retained.  Wallet 4 does not link the Favorites entires to the actual entries.  Only the name of the item is referenced in the favorites list, but this name may not be unique, so it is not possible to tag an item as a favorite.
```
Comment (sjkx) [inline convert_to_1p4/README.md]: s/entires/entries/

File: convert_to_1p4/README.md [hunk 2]
```diff
@@ -564,11 +603,11 @@ You may also want to export your Identities and Safenotes now, using a similar p
 
 You may now quit RoboForm.  Perform conversions and imports on each of the files you exported.  The **roboform** converter will convert one or more export files at once, so you may specify each of the files you exported above on the same command line.  Example:
 
-    perl convert_to_1p4 roboform -v pm_export_logins.html pm_export_identities.html pm_export_safenotes.html
+    perl convert_to_1p4.pl roboform -v pm_export_logins.html pm_export_identities.html pm_export_safenotes.html
     
 The command above will convert all three export files into a single 1PIF file containing your converted logins, identity items, and safenotes.
 
-Note: RoboForm version 7 and above does not export the full original URL for Login items.  You will probably need to edit some login entires within 1Password to make Open and Fill work correctly.
+**Note**: RoboForm version 7 and above does not export the full original URL for Login items.  You will probably need to edit some login entires within 1Password to make Open and Fill work correctly.
```
Comment (sjkx) [inline convert_to_1p4/README.md]: s/entires/entries/

**Comment by sjkx** [comment]
Are you intending to review/merge this, @robyoder?

**Comment by mikecappella** [comment]
Got an email from Rob on Friday... said he would address this shortly.

**Pull request merged**